<?xml version="1.0" encoding="UTF-8"?>
<launch>

    <node pkg="camera_lidar_calibrator" type="camera_lidar_calibrator_node" name="camera_lidar_calibrator_node" output="screen" required="true">

        <!-- BAG -->
        <param name="sensor_type" value="VLP16" />
        <param name="bag_path" value="exemplaryData/stereo_velo/stereo_velo.bag" />
        <param name="read_countours_path" value="exemplaryData/stereo_velo/laser_detection.txt" />

        <!-- LEFT -->
        <param name="camera_topic" value="/stereo/left/image_raw/compressed" />
        <param name="read_chessboards_path" value="exemplaryData/stereo_velo/stereo_left.txt" />

        <param name="fx" value="594.33352642" />
        <param name="fy" value="593.65115891" />
        <param name="cx" value="355.23489022" />
        <param name="cy" value="271.72366991" />

        <param name="d0" value="-0.24238145" />
        <param name="d1" value="0.10238686" />
        <param name="d2" value="-0.00017" />
        <param name="d3" value="-0.00004358" />


        <!-- PARAMS -->
        <param name="read_countours_from_file" value="true" />
        <param name="read_chessboards_from_file" value="true" />

        <param name="cloud_every_n" value="1" />
        <param name="image_every_n" value="1" />

        <param name="time_shift_fixed" value="false" />
        <param name="artifical_time_shift" value="0.00" />

        <param name="intelligent_edge_selection" value="false" />
        <param name="max_edge_number" value="10000000" />

        <param name="increase_point_number" value="0" />
    </node>
</launch>